src/pages/folder/Index.tsx

```diff
@@ -1,53 +1,179 @@
+      return;
+    }
+    // api/users/1/links?folderId={해당 폴더 ID}
+    setDynamicAPI(`${FOLDERCONTANTLISTAPI}?folderId=${api}`);
+    const result = menu?.data.filter((data) => +data.id === +api);
+    result && setTitle(result[0]?.name as "");
+  };
+
+  useEffect(() => {}, [menu, contant]);
 
   return (
     <>
       <ContainHead>
-        <FolderContainHeadInner>
-          <img src={logo} alt="코드잇 로고" />
-          <p>@코드잇</p>
-          <TitleMs as={"h2"}>⭐️ 즐겨찾기</TitleMs>
-        </FolderContainHeadInner>
+        <LinkAddHeadInner>
+          <Input
+            $inputClass={"input__link--add"}
+            $placeholder={"링크를 추가해 보세요"}
+            $beforeBgIcon={link}
+            $btnShow={true}
+            $btnText={"추가하기"}
+            $btnClass={"button--gradient mideum"}
+          />
+        </LinkAddHeadInner>
       </ContainHead>
       <ContainBody>
-        <FolderContainBodyInner>
-          <SearchInput></SearchInput>
-          {isLoading ? (
-            <PostCardWrap>
-              {cardInfo
-                ? cardInfo.map((data) => <PostCard key={data.id} {...data} />)
-                : "리스트가 없습니다."}
-            </PostCardWrap>
+        <BodyInner>
+          {/* 검색창 */}
+          <BoxLinkSearch>
+            <Input
+              $inputClass={"input__link--search"}
+              $placeholder={"링크를 검색해 보세요."}
+              $beforeBgIcon={search}
+            />
+          </BoxLinkSearch>
+          {/* 폴더버튼 리스트 */}
+          <BookmarkBox>
+            <BookMarkBtnList>
+              {menuLoading && (
+                <>
+                  <Button
+                    $id={"all"}
+                    $btnClass={`button--outlined ${
+                      btnActive === -1 ? "active" : ""
+                    }`}
+                    clickEvent={handleClick}
+                    $clickEventName={"bookmarkId"}
+                    $clickIndex={-1}
+                  >
+                    전체
+                  </Button>
+                  {menu &&
+                    menu.data.map((menu: any, i) => (
+                      <Button
+                        key={menu.id}
+                        $id={menu.id}
+                        $btnClass={`button--outlined ${
+                          btnActive === i ? "active" : ""
+                        }`}
+                        clickEvent={handleClick}
+                        $clickEventName={"bookmarkId"}
+                        $clickIndex={i}
+                      >
+                        {menu.name}
+                      </Button>
+                    ))}
+                </>
+              )}
+            </BookMarkBtnList>
+            <div>
+              <Button $btnClass={"button--icon-after"} $afterButtonIcon={add}>
+                폴더추가
+              </Button>
+            </div>
+          </BookmarkBox>
+          {/* 버튼 수정 */}
+          <ShareBox>
+            <SubTitle>{title}</SubTitle>
+            {title === "전체" || (
```

Comment:
|| 로 하면 첫번째 인자가 true 이면 첫 번째 인자를 반환하고, false 이면 두 번째 인자를 반환합니다!
title 이 전체일 때 뒤의 내용이 렌더링되길 원하신다면 && 이 맞을 거 같습니다
```suggestion
            {title === "전체" && (
```